ng-dev/ts-circular-dependencies/analyzer.ts

```diff
@@ -32,13 +33,18 @@ const DEFAULT_EXTENSIONS = ['ts', 'js', 'd.ts'];
 export class Analyzer {
   private _sourceFileCache = new Map<string, ts.SourceFile>();
 
+  private _ignoreTypeOnlyChecks: boolean;
+
   unresolvedModules = new Set<string>();
   unresolvedFiles = new Map<string, string[]>();
 
   constructor(
     public resolveModuleFn?: ModuleResolver,
+    ignoreTypeOnlyChecks: boolean = false,
     public extensions: string[] = DEFAULT_EXTENSIONS,
-  ) {}
+  ) {
+    this._ignoreTypeOnlyChecks = !!ignoreTypeOnlyChecks;
```

Comment:
Now you can also spare the extra assignment and member declaration by just marking it e.g. as private in the constructor (that was mostly why I preferred the simple parameter)